__tests__/Unit/Components/Tasks/Card.test.tsx

```diff
@@ -231,6 +233,60 @@ describe('Task card', () => {
         expect(closeTaskBtn).toBeInTheDocument();
     });
 
+    it('Should make PATCH call with feature flag', function () {
+        // const mockFetch = jest.mock('fetch');
+        function Wrapper({
+            children,
```

Comment:
can we test if we don't pass feature flag then what will happen to the patch call

Reply:
Yeah the above test written by @bharati-21 doesn't pass the feature flag and its working as feature flag is optional.